truss/patch/calc_patch.py

```diff
@@ -45,68 +56,72 @@ def calc_truss_patch(
     )
     # TODO(pankaj) Calculate model code patches only for now, add config changes
     # later.
+
     truss_spec = TrussSpec(truss_dir)
-    model_module_path = str(truss_spec.model_module_dir.relative_to(truss_dir))
-    training_module_path = str(truss_spec.training_module_dir.relative_to(truss_dir))
+    model_module_path = _relative_to_root(truss_spec.model_module_dir)
+    data_dir_path = _relative_to_root(truss_spec.data_dir)
+    bundled_packages_path = _relative_to_root(truss_spec.bundled_packages_dir)
+
+    def _under_unsupported_patch_dir(path: str) -> bool:
+        """
+        Checks if the given path is under one of the directories that don't
+        support patching. Note that if path `is` one of those directories that's
+        ok, because those empty directories can be ignored from patching point
+        of view.
+        """
+        return _strictly_under(path, [data_dir_path, bundled_packages_path])
 
     patches = []
     for path in changed_paths["removed"]:
         if path.startswith(model_module_path):
-            relative_to_model_module_path = str(
-                Path(path).relative_to(model_module_path)
-            )
             patches.append(
                 Patch(
                     type=PatchType.MODEL_CODE,
                     body=ModelCodePatch(
                         action=Action.REMOVE,
-                        path=relative_to_model_module_path,
+                        path=_relative_to(path, model_module_path),
                     ),
                 )
             )
-        elif path.startswith(training_module_path):
-            # Ignore training changes from patch
-            continue
-        else:
+        elif path == CONFIG_FILE:
+            # Don't support removal of config file
+            logger.info(f"Patching not supported for removing {path}")
```

Comment:
I think `logger.warn` is more appropriate here since delete config is a problem for everything